### ISSUE TITLE
model KVStore initial migration not in package when installing via pip

### ISSUE THREAD

**Issue by jlariza** [mariocesar/sorl-thumbnail#458]
I installed the package via pip and I was getting a programming error. I had to create the KVStore inital migration manually to make it work because apparently it was not there.

python manage.py makemigrations thumbnail

Migrations for 'thumbnail':
  0001_initial.py:
    - Create model KVStore

It should be included in the package to avoid problems. thank you

**Comment by kevin-brown** [comment]
It already exists.

https://github.com/mariocesar/sorl-thumbnail/blob/c7d9e9d6d55d89f0a7aaa867d087d713cd76937c/sorl/thumbnail/migrations/0001_initial.py

It's just not in any releases on PyPI, which is the real issue.

**Comment by wiesson** [comment]
Any updates to this issue?

**Comment by dmwyatt** [comment]
@mariocesar This really needs addressed!  No one can use this library without googling around and figuring out that it's not working because migrations aren't included in the package they installed.

**Comment by mariocesar** [comment]
@dmwyatt You are right, need wrap a plan to maintain some compatibility and release a new version in the process.

If you want to help, solving tests and maintaining compatibility while doing the upgrade will fix all we need to release.

**Comment by pembeci** [comment]
Installed with pip, followed the documentation, decided to use the default settings and bumped into this. Easy to solve of course but for a beginner user this still seems to be an issue.

**Comment by mariocesar** [comment]
Main issue is that the stable version is really outdated, and the work on the next released is stalled. If someone wants to give a hand on the next release is welcome. There is really not much work, update some tests, rename somethings, and other minor things

**Comment by pembeci** [comment]
@mariocesar, thanks for the update. Is there a todo list for the new release somewhere so newcomers can try to help?

**Comment by hobarrera** [comment]
Can we at least have a release `12.3.1` branched off with 12.3, but with just the migration included? Currently, bootstrapping most projects has hacks in place to generate that migration (or `curl` it from github), and it's really hacky.

Also, like @pembeci said, a list of things that need work for the next release (maybe just tag the GH issues) would be great. I'm sure I can make the time to work on them at some point, but most importantly, I'm sure other are.
